api/api.pb.txt

```diff
@@ -4065,6 +4077,14 @@ file {
         oneof_index: 0
         json_name: "registry"
       }
+      field {
+        name: "Config"
```

Comment:
@dperny I noticed this field uses an uppercase `c`, whereas the other fields are lowercase; was there a specific reason for this, or can we change that to be consistent with the other ones?

Reply:
Fixed in https://github.com/docker/swarmkit/pull/2813